### PR TITLE
Rename "What can I do about Bufferbloat?" 

### PR DESCRIPTION
**Pull request by richb-hanover**
(from "What to do about Bufferbloat?") and also

Update Make Wi-Fi Fast home page with new information about LEDE wifi fixes and Toke's paper

[This time for sure...]

### CODE REVIEW

File: content/bloat/wiki/What_can_I_do_about_Bufferbloat.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
 ---
-# What to do about Bufferbloat?
+# What Can I Do About Bufferbloat?
 
 Bufferbloat is high latency (or lag) that occurs when there's other
-traffic on your network. Use the [DSLReports Speed
-Test](http://dslreports.com/speedtest) or run one of the [Tests for Bufferbloat](Tests_for_Bufferbloat.md) to see if it's present.
+traffic on your network. 
+Use the [DSLReports Speed Test](http://dslreports.com/speedtest) 
+or run one of the [Tests for Bufferbloat](/bloat/wiki/Tests_for_Bufferbloat.md) to see if it's present.
```
Comment (tohojo) [inline content/bloat/wiki/What_can_I_do_about_Bufferbloat.md]: Why the change of link style?

File: content/bloat/wiki/What_can_I_do_about_Bufferbloat.md [hunk 2]
```diff
@@ -18,7 +21,7 @@ will need to find a way to fix the **router.**
 
 ## How Can I Tell if My Router Has Bufferbloat?
 
--   Use [DSL Reports Speed Test](http://dslreports.com/speedtest) or any of the other tests on [Tests for Bufferbloat](Tests_for_Bufferbloat.md)
+-   Use [DSL Reports Speed Test](http://dslreports.com/speedtest) or any of the other tests on [Tests for Bufferbloat](/bloat/wiki/Tests_for_Bufferbloat/)
```
Comment (tohojo) [inline content/bloat/wiki/What_can_I_do_about_Bufferbloat.md]: Also changed link style (to absolute path)

File: content/cerowrt/wiki/What_t.md [hunk 3]
```diff
@@ -8,4 +8,4 @@ What t
 ======
 
 Redirect to
-http://www.bufferbloat.net/projects/cerowrt/wiki/What\_to\_do\_about\_Bufferbloat
+http://www.bufferbloat.net/projects/bloat/wiki/What\_can\_I\_do\_about\_Bufferbloat
```
Comment (tohojo) [inline content/cerowrt/wiki/What_t.md]: Not sure what this page is supposed to be doing?

File: content/make-wifi-fast/wiki/index.md [hunk 4]
```diff
@@ -1,13 +1,34 @@
...
+
+We believe that the same sort of systems thinking that went on in the Bufferbloat Project can lead to 
+performance improvements of an order of magnitude or more in Wi-Fi.
+
+## Current Status
+
+As of early 2017, we have achieved many of these improvements, specifically a decrease of latency by at least an order of magnitude, with fair sharing of air time across fast and slow devices.
+
+* Working software is available in <a href="https://lede-project.org" target="blank">LEDE firmware</a> that runs on off-the-shelf routers, x86 boxes, and embedded systems.
+* An academic paper has been published to describe the current state of the working software,
```
Comment (tohojo) [inline content/make-wifi-fast/wiki/index.md]: Hasn't technically been published yet; only the preprint...

File: layouts/index.html [hunk 5]
```diff
@@ -14,29 +14,29 @@
...
-            <ul>
-              <li><strong><a href="/bloat/wiki/What_to_do_about_Bufferbloat/">What to do about Bufferbloat?</a></strong> If you have bad latency/lag, or someone has told you there is Bufferbloat in your network. This page lists several steps that you can take to measure the bloat in your network, and how make solve it completely.</li>
+              <ul>
+              <li><strong><a href="http://localhost:1313/projects/bloat/wiki/What_can_I_do_about_Bufferbloat/">What Can I Do About Bufferbloat?</a></strong> 
+              If you have bad latency/lag, or someone has told you there is Bufferbloat in your network, 
+              this page lists several steps for measuring the bloat in your network, and for solving it completely.</li>
 
-              <li><strong><a href="/bloat/wiki">The Bufferbloat Project</a></strong> 
-              has largely addressed
+              <li><strong><a href="http://localhost:1313/projects/bloat/wiki">
```
Comment (tohojo) [inline layouts/index.html]: As is this

File: layouts/index.html [hunk 6]
```diff
@@ -14,29 +14,29 @@
               projects provide a webspace for addressing chaotic and laggy
               network performance. We have a number of projects in flight:</p>
 
-            <ul>
-              <li><strong><a href="/bloat/wiki/What_to_do_about_Bufferbloat/">What to do about Bufferbloat?</a></strong> If you have bad latency/lag, or someone has told you there is Bufferbloat in your network. This page lists several steps that you can take to measure the bloat in your network, and how make solve it completely.</li>
+              <ul>
+              <li><strong><a href="http://localhost:1313/projects/bloat/wiki/What_can_I_do_about_Bufferbloat/">What Can I Do About Bufferbloat?</a></strong>
```
Comment (tohojo) [inline layouts/index.html]: This is certainly broken

File: layouts/index.html [hunk 7]
```diff
@@ -14,29 +14,29 @@
...
-              years. These algorithms have been deployed in millions of
-              computers, and reduce the induced delay from competing traffic on
-              a bottleneck link to the order of 20 msec.</li>
+              The cake algorithm (and its predecessors CoDel and fq_codel algorithms) are the first 
+              fundamental advances in the state of the art of network Active Queue Management in many, many years. 
+              These algorithms have been deployed in millions of computers, and reduce the induced 
+              delay from competing traffic on a bottleneck link to the order of 20 msec.</li>
 
-              <li><strong><a href="/make-wifi-fast/wiki">
+              <li><strong><a href="http://localhost:1313/projects/make-wifi-fast/wiki">
```
Comment (tohojo) [inline layouts/index.html]: And this

File: layouts/index.html [hunk 8]
```diff
@@ -14,29 +14,29 @@
...
               same team members as the Bufferbloat project, intends to improve
               Wi-Fi's speed and use of the spectrum by inserting CoDel/fq_codel
               into the Wi-Fi queues, and actively measuring the power required
               for successful transmission, in order to minimize contention and
               interference on the RF channel. 
-              As of early 2017, our efforts to remove queues, and add Airtime Fairness 
-              to Wi-Fi stacks is bearing fruit, and is available in 
-              <a href="https://lede-project.org" target="blank">LEDE firmware.</a> </li>
+              As of early 2017, our efforts to remove queueing latency and add Airtime Fairness 
+              to Wi-Fi stacks have borne fruit. See the <a href="http://localhost:1313/projects/make-wifi-fast/wiki/">Make Wi-Fi Fast Status</a> page.
```
Comment (tohojo) [inline layouts/index.html]: And this...

File: content/bloat/wiki/What_can_I_do_about_Bufferbloat.md [hunk 9]
```diff
@@ -1,14 +1,17 @@
 ---
-title: What to do about Bufferbloat?
-date: 2017-02-10T11:30:12
-lastmod: 2017-02-10T11:30:12
+title: What Can I Do About Bufferbloat?
+date: 2017-03-10T09:10:12
+lastmod: 2017-03-10T09:20:12
 type: wiki
+aliases:
+    - /projects/bloat/what_to_do_about_bufferbloat.md
```
Comment (tohojo) [inline content/bloat/wiki/What_can_I_do_about_Bufferbloat.md]: There shouldn't be an .md suffix on the alias, and I think it is case sensitive...

**Comment by richb-hanover** [comment]
Hi Toke,

1) I don't understand the rules/ethics/politics of publishing a link to a preprint. I look to your guidance, but we shouldn't hide our light under a bushel...
2) I changed several sentences to start each on a new line. It makes detecting editing changes much easier to see (in the future).
3) Fixed the absolute links that could be relative.
4) The "What_t.md" file is a holdover from the Redmine wiki. (Its title started out to be "What to do about Bufferbloat", but something failed, and I didn't see how to modify the title.) I've deleted it.
5) ... http://localhost:1313 <http://localhost:1313/> ... Stupid. Fixed.
6) Fixed many aliases to the renamed "What can I do" - I think their case-sensitivity and suffix are correct for links that are in the wild. (Is it possible to browse the error.log file on the bufferbloat.net <http://bufferbloat.net/> server?)

Thanks for your patience.

Rich

**Comment by tohojo** [comment]
For the paper I would just write something like "An academic paper describing the current state of the working software has been submitted for publication; a preprint is available at..."

I'll email you the log file.

**Comment by richb-hanover** [comment]
OK. I'll update the description in my branch. Does it otherwise look OK?

Thanks.

Rich

**Comment by richb-hanover** [comment]
I just pushed that update. Please check.

**Comment by tohojo** [comment]
There are some weird changes to cerowrt/issues/216.md - and a large diff of a .csv file. Why is that?

Also, aliases should not have .md extensions; they operate solely on the publicly visible URL space...

**Comment by richb-hanover** [comment]
* There don't seem to be any urls-with-.md in the wild, so I have removed those.  

* The 216.md file has wacky email addresses (if anyone attempted to use them) that were tripping up LinkChecker. Same for the .csv file. But I care more about publishing these changes than being error-free.

* I added a link to the Untangle firewall (it has fq_codel) and fixed lots of wording/typos.

**Comment by dtaht** [comment]
toke is on a big boat somewhere, and the division of labor here is unclear. But LGTM.

**Comment by richb-hanover** [comment]
Hey! The magic works!
